qopt_best_practices/swap_strategies/build_circuit.py

```diff
@@ -116,19 +116,27 @@ def apply_qaoa_layers(
 
 
 def create_qaoa_swap_circuit(
-    cost_operator: SparsePauliOp,
+    cost_operator: list[tuple[str, float]],
```

Comment:
Why this change? Looking at the code below you call `cost_operator.paulis`. Therefore it seems that the type hint should be on `SparsePauliOp` and not `list[tuple[...`

Reply:
Agreed, I changed this now.